test/shopify-cli/theme/syncer_test.rb

```diff
@@ -757,6 +758,47 @@ def test_broken_file_when_file_is_not_broken
         refute(@syncer.broken_file?(file))
       end
 
+      # Case where file is an asset but has errors, resp is: { errors => { asset => ["..."]}}
+      def test_parse_valid_theme_asset_with_errors_bulk
```

Comment:
Case where we have a valid theme asset, but there are errors with it. Error comes in form:
{errors => {asset => [..] || "..."}}